sympy/parsing/sympy_parser.py

```diff
@@ -931,11 +931,29 @@ def visit_BinOp(self, node):
         if node.op.__class__ in self.operators:
             sympy_class = self.operators[node.op.__class__]
             right = self.visit(node.right)
-
+            left = self.visit(node.left) 
+            if isinstance(node.left, ast.UnaryOp) and (isinstance(node.right, ast.UnaryOp) == 0) and sympy_class in ('Mul'):
```

Comment:
I *think* you might want to add a comma after "Mul" as in `('Mul',)` otherwise ('Mul') is simply a string with redundant parentheses and if we ever get an "M" class or a "Mu" class this test could fail ;-)